### PR TITLE
Posts/2022 23

### PR DESCRIPTION
**Pull request by ntlhui**

### CODE REVIEW

**Comment by ntlhui** [comment]
This includes all latest posts

File: _posts/2023-08-14-2023-summer-research-students.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  photo: assets/people/d.mathews.png
+---
+
+This summer, we are hosting 19 students from around the US in our 2023 summer research program at UC San Diego.  We'll be working on measuring fish length with lasers, building Smartfins, classifying bird calls, and tracking radio signals.  Meet our students by reading their bios below:
+
+## Student Researchers
+{% for student in page.students %}
+<div class="floatclearfix">
+    {% if student.photo %}
+    <p><a href="{{ student.photo | absolute_url }}"><img alt="{{student.name}}" class="{{student.align}}" src="{{student.photo | resize: '180x240'}}"></a></p>{% endif %}
```
Comment (Sean1572) [inline _posts/2023-08-14-2023-summer-research-students.md:60]: change to `<p style="display: float; justify-content: center;">`

File: _posts/2022-08-17-2022-summer-research-students.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+- bio: Robert O'Brien is a rising junior at Princeton University studying electrical and computer engineering. This summer he mainly focused on improving firmware for the Smartfin team. Additionally, he made improvements in Smartfin's hardware and fin manufacturing techniques. Robert hopes to continue working with embedded systems and conservation after he graduates. Robert likes to surf and play rugby, however back in his hometown in Virginia there aren't many waves.
+  photo: assets/people/r.obrien.jpg
+---
+This summer, we are hosting 12 students from around the US in our 2022 summer research program, finally back in person at UC San Diego.  We'll be working on measuring fish length, building SmartFins, labelling bird calls, and tracking animals.  Meet our students by reading their bios below:
+
+## Student Researchers
+{% for student in page.students %}
+<div class="floatclearfix">
+    {% if student.photo %}
+    <p><a href="{{ student.photo | absolute_url }}"><img alt="{{student.name}}" class="{{student.align}}" src="{{student.photo | resize: '180x240'}}"></a></p>{% endif %}
```
Comment (Sean1572) [inline _posts/2022-08-17-2022-summer-research-students.md:43]: see 2023 summer reu student page

**Review by Sean1572 — CHANGES_REQUESTED** [review]

File: _posts/2022-08-17-2022-summer-research-students.md (hunk 2 above)
Comment (ntlhui) [reply _posts/2022-08-17-2022-summer-research-students.md]: Fixed in 62cd4a688e0dc2b4e1ed05978c53e12fe7d08207

**Review by Sean1572 — APPROVED** [review]

**Pull request merged**